index.d.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Shows the cursor.
+ * 
+ * @param stream (Default: process.stderr)
```

Comment:
There should be a `-` here. See the style guide. Also see the style guide regarding `Default` formatting.